packages/static-wado-creator/lib/mkdicomwebConfig.js

```diff
@@ -99,6 +95,11 @@ const { mkdicomwebConfig } = ConfigPoint.register({
         description: "expand bulkdata relative uri to use full relative path (should also be set when using --prepend-bulk-data-uri)",
         defaultValue: false,
       },
+      {
```

Comment:
This is the new correct location for this argument, along with teh way to read the value from the correct configuration file.